pkg/events/ftrace.go

```diff
@@ -0,0 +1,348 @@
+package events
+
+import (
+	"os"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	lru "github.com/hashicorp/golang-lru/v2"
+
+	"github.com/aquasecurity/tracee/pkg/counter"
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+const (
+	sysKernelPrefix      = "/sys/kernel/"
+	sysKernelDebugPrefix = sysKernelPrefix + "debug/"
+)
+
+const (
+	symbolIndex = iota
+	trampIndex
+	callbackFuncIndex
+	callbackOffsetIndex
+	callbackOwnerIndex
+	flagsIndex
+	countIndex
+)
+
+var (
+	reportedFtraceHooks *lru.Cache[string, []trace.Argument]
+)
+
+func init() {
+	var err error
+	reportedFtraceHooks, err = lru.New[string, []trace.Argument](2048)
+	if err != nil {
+		panic(err)
```

Comment:
So this is exactly what it is doing - I panic and it will be caught, logged and logic will stop.
If allocation failed no point in running logic.
I'll add a comment mentioning the above. 
Sounds good?